### DISCUSSION TITLE
Can I source code for demo page?

### DISCUSSION THREAD

**Discussion by qasims** [generic-ui/generic-ui#30] [Q&A]
Can I get source code for https://generic-ui.com/demo grid?

**Comment by generic-ui** [comment]
We will try to create a separate repository with it. Please give us some time.

**Reply by jmmiclet** [reply]
Hello,
Can we have repository for https://generic-ui.com/examples/dynamic-data?
create-project.request source code is missing in the page.
Thank you

**Reply by kpawelczak** [reply]
hello, @jmmiclet we have updated the example page: https://generic-ui.com/examples/dynamic-data . Additionally you can look at the Stackblitz: [Link](https://stackblitz.com/edit/angular-ivy-atbshd?file=src/app/app.component.ts) or repository on github: [Link](https://github.com/kpawelczak/gui-grid-example-dynamic-data)

**Reply by jmmiclet** [reply]
Thanks, it's a great help.

**Comment by generic-ui-dr** [comment]
hey @qasims we have prepared [stackblitz ](https://stackblitz.com/edit/angular-ivy-qkjzcs?file=angular.json,src%2Fapp%2Fdemo%2Fdemo-project.component.ts)demo and github [repository](https://github.com/generic-ui-dr/gui-grid-demo)
